### ISSUE TITLE
Change namespace and re-release on packagist

### ISSUE THREAD

**Issue by neverything** [verze-app/solana-php-sdk#46]
As mentioned in https://github.com/verze-app/solana-php-sdk/discussions/36#discussion-4027681, we will change the namespace to `Verze` and re-release this package on packagist.

- [ ] Update namespace
- [ ] Update tests
- [ ] Update documentation
- [ ] Update roadmap
- [ ] Check with Tighten to remove the old package
- [ ] Release new version of the package

Any help with this is greatly appreciated as my time is a bit limited currently.

**Comment by chongkan** [comment]
@neverything I asume this task list is outdate and Verze is no longer the target namespace as of 2023?

**Comment by neverything** [comment]
@chongkan this is correct, as we don't work on this project any longer, I can transfer it to you if you want or you can start on a fork.
